### DISCUSSION TITLE
Grupo de Teams!

### DISCUSSION THREAD

**Discussion by AranxaMo** [Launch-X-Latam/MisionFrontEnd#438] [Team Posts]
Hola chicos! Solo para decirles que después de que hayan mandado su solicitud al grupo de Teams sean pacientes, no tardan mucho en agregarlos. Yo mande la solicitud el día de ayer por la noche y hoy ya me apareció el grupo en Teams. Para que se pongan al día con los temas atrasados 😄

**Comment by FernandaTre** [comment]
Que grupo de temas? 😱

**Comment by gumodi35** [comment]
Los primeros temas son sobre el diseño de 🥤 aplicaciones, toma de requerimientos y definir tu público objetivo y tú buyer persona... Hoy fue un vistazo de HTML, 💻 pero relax, hay tiempo para poder hacerlo, esto no es una 🏃 carrera... Cualquier duda existe está comunidad para ayudar, 🆘  esperen su acceso, vean las sesiones y vayan 🎦 haciendo la práctica, 🧑‍💼 lo más importante es que no se desesperen. 🐇

**Comment by sudoagile** [comment]
Creo que este es el contenido, ahí están los videos por fechas

https://web.microsoftstream.com/user/96eeb66f-b984-4083-8ea6-74c34a07a34a



De: Gualberto ***@***.***>
Fecha: viernes, 7 de octubre de 2022, 22:27
Para:
CC: Subscribed ***@***.***>
Asunto: Re: [Launch-X-Latam/mision-frontend] Grupo de Teams! (#7)

Los primeros temas son sobre el diseño de 🥤 aplicaciones, toma de requerimientos y definir tu público objetivo y tú buyer persona... Hoy fue un vistazo de HTML, 💻 pero relax, hay tiempo para poder hacerlo, esto no es una 🏃 carrera... Cualquier duda existe está comunidad para ayudar, 🆘 esperen su acceso, vean las sesiones y vayan 🎦 haciendo la práctica, 🧑‍💼 lo más importante es que no se desesperen. 🐇

**Comment by sudoagile** [comment]
Este si creo que es

https://web.microsoftstream.com/group/1cd6a6ad-7b5f-4845-b108-4f1ccb16d8dd?view=videos


De: Gualberto ***@***.***>
Fecha: viernes, 7 de octubre de 2022, 22:27
Para:
CC: Subscribed ***@***.***>
Asunto: Re: [Launch-X-Latam/mision-frontend] Grupo de Teams! (#7)

Los primeros temas son sobre el diseño de 🥤 aplicaciones, toma de requerimientos y definir tu público objetivo y tú buyer persona... Hoy fue un vistazo de HTML, 💻 pero relax, hay tiempo para poder hacerlo, esto no es una 🏃 carrera... Cualquier duda existe está comunidad para ayudar, 🆘 esperen su acceso, vean las sesiones y vayan 🎦 haciendo la práctica, 🧑‍💼 lo más importante es que no se desesperen. 🐇
